### DISCUSSION TITLE
How to specify a display area for many sprites?

### DISCUSSION THREAD

**Discussion by Aqaao** [bevyengine/bevy#14092] [Q&A] [answered]
Sprite will not be visible when it leaves area. Cropped when sprite is close to the edge. 

Camera.viewport does the job but I need to keep tweaking it when changing window resolution. Is there a simpler way?

**Comment by miketwenty1** [comment]
I haven't had to deal with this problem myself yet, but I'd check out:
```rust
pub visible_entities: VisibleEntities,
pub frustum: Frustum 
```
https://docs.rs/bevy/0.13.2/bevy/render/view/struct.VisibleEntities.html
https://docs.rs/bevy/0.13.2/bevy/render/primitives/struct.Frustum.html

Within the Camera2dBundle

**Reply by Aqaao** [reply]
Thank your answer. But them doesn't seem to crop sprite, sprite just disappeared.

**Reply by miketwenty1** [reply, marked as answer]
The only way I would know to "crop" a sprite on screen would be to make it visible but then adjust the camera so it appears to be cropped, or make a high z layer mesh/sprite or ui that will cover it up as it leaves an area. Do you have a screenshot of the problem so it's more clear?

**Reply by Aqaao** [reply]
In the end I achieved this effect using a background with a transparent mask, which for me was not the best solution because my graphic assets have complex layer relationship. anyway thanks for your help!
